src/anm2.py

```diff
@@ -150,6 +150,7 @@ def render(self):
 
             # Load the Image
             qimg = QImage(imgPath)
+            #qimg.save(imgPath)
```

Comment:
should we delete this line?

Reply:
I used this to fix the formats so I left it in if I need to do it again another time.